src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -116,7 +116,7 @@ public Task Stats()
         [RavenAction("/databases/*/timeseries", "GET", AuthorizationStatus.ValidUser)]
         public async Task Read()
         {
-            var documentId = GetStringQueryString("id");
+            var documentId = GetStringQueryString("docId");
             var names = GetStringValuesQueryString("name");
```

Comment:
Shouldn't it be `"names"`?

Reply:
No, because you usually specify just a single one, and add it multiple times